### PR TITLE
Package management (after end of release 35)

### PR DESCRIPTION
**Pull request by nytamin**
This is an PR that contains the Package-Manager related changes up to the end of Release 35.

It consists mainly these changes:

* The listen-to-packageInfo feature, originally implemented as a quick-fix for release 32.
* Moving of PackageInfo interfaces into the blueprints-integration package

I'd like to merge this into Release36 asap, in order to keep reviews and our effort to keep the branches in sync minimal.

### CODE REVIEW

File: meteor/server/api/ingest/packageInfo.ts [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import { PackageInfoDB } from '../../../lib/collections/PackageInfos'
+import { RundownId, Rundowns } from '../../../lib/collections/Rundowns'
+import { SegmentId } from '../../../lib/collections/Segments'
+import { assertNever, lazyIgnore } from '../../../lib/lib'
+import { logger } from '../../logging'
+import { runIngestOperationWithCache } from './lockFunction'
+import { Meteor } from 'meteor/meteor'
+import { regenerateSegmentsFromIngestData } from './generation'
+
+/** to-be @deprecated hack for backwards-compatibility*/
```
Comment (sparkpunkd) [inline meteor/server/api/ingest/packageInfo.ts:12]: In or out? TODO or to-deprecate? Which is it?

File: meteor/server/api/ingest/packageInfo.ts [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+			for (const packageId of packageIds) {
+				const pkg = cache.ExpectedPackages.findOne(packageId)
+				if (pkg) {
+					if (
+						pkg.fromPieceType === ExpectedPackageDBType.PIECE ||
+						pkg.fromPieceType === ExpectedPackageDBType.ADLIB_PIECE
+					) {
+						segmentsToUpdate.add(pkg.segmentId)
+					}
+					// TODO - trigger baseline regenerateion
```
Comment (sparkpunkd) [inline meteor/server/api/ingest/packageInfo.ts:131]: Does this TODO neded to be done? Sounds important?

**Review by sparkpunkd — APPROVED** [review]
A couple of TODOs that might need addressing - see separate comments - but overall this is a good way of introducing package manager features consistent with the mono-repo.

A number of these features were tested in a broadcast trial using a branch off R32 so we can be fairly confident in using this release for news loops with packages going forward.

File: meteor/server/api/ingest/packageInfo.ts (hunk 1 above)
Comment (sparkpunkd) [reply meteor/server/api/ingest/packageInfo.ts:12]: Julian and I decided to remove this method for #525 ... see commit https://github.com/nrkno/tv-automation-server-core/pull/525/commits/0301a5e987ff6d8836e9f0aeb04c61834582b37b

File: meteor/server/api/ingest/packageInfo.ts (hunk 2 above)
Comment (nytamin) [reply meteor/server/api/ingest/packageInfo.ts]: We're leaving a stub for now, to move forward. will implement properly later.

**Pull request merged**